.mvn/settings.xml

```diff
@@ -1,16 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
-          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
```

Comment:
let's add this into a root pom.xml? No need to make it more complicated by putting itinto settings.xml

Reply:
We had an offline discussion and decide to keep it for now, as putting SETTINGS and POM namespace together could cause problems.